### PR TITLE
Add "asciinema Player for GitHub"

### PR DESCRIPTION
**Pull request by plibither8**
A small extension I made to make it more convenient to view asciinema screencasts on GitHub :slightly_smiling_face:

-----
[View rendered README.md](https://github.com/plibither8/awesome-browser-extensions-for-github/blob/patch-1/README.md)

### CODE REVIEW

**Comment by stefanbuck** [comment]
Yeah that's awesome.

File: README.md [hunk 1]
```diff
@@ -634,6 +634,15 @@ ZenHub is the first and only project management suite that works natively within
  ![ZenHub](https://dl.dropboxusercontent.com/s/yosmyg8zsl5tyc5/zenhub.jpg)
 </details>
 
+## asciinema Player for GitHub <a href="https://chrome.google.com/webstore/detail/asciinema-player-for-gith/mkllmpcfhjcbkmegdpnaemhpdddffhil"><img src="https://raw.githubusercontent.com/alrra/browser-logos/master/src/chrome/chrome_48x48.png" width="24" /></a>
```
Comment (stefanbuck) [inline README.md]: Please wrap the title with a link to your repo. That way people can easily find the source. Thanks

Comment (plibither8) [reply README.md]: Sure thing, done :)

Comment (stefanbuck) [reply README.md]: Thank you

**Comment by plibither8** [comment]
Thanks :tada:!

**Pull request merged**